src/Illuminate/Foundation/Http/HtmlDumper.php

```diff
@@ -133,10 +133,11 @@ protected function getDumpSourceContent()
         $source = sprintf('%s%s', $relativeFile, is_null($line) ? '' : ":$line");
 
         if ($editor = $this->editor()) {
+            $editorProjectPath = $this->editorProjectPath();
```

Comment:
There is no way to infer the project's path, so the user don't have to hardcode it on his configuration file / environment variables?

Reply:
Yes, there is no way to automatically infer the project's path as the editor that the users use.
That's why I have the user hardcode it.

Unfortunately, I have no idea.
Do you have any solution?

Reply:
@fabio-ivona 
It sounds good!
`IGNITION_EDITOR` and `IGNITION_LOCAL_SITES_PATH ` is suitable for them.

Reply:
I would use a more general key, like (EDITOR_BASE_PATH) and ask to Spatie to support that one as well

using "IGNITION" for this feature doesn't seem consistent to me